Library/Formula/nspr.rb

```diff
@@ -32,12 +34,12 @@ def install
         --enable-strip
         --enable-pthreads
         --enable-ipv6
-        --enable-macos-target=#{MacOS.version}
       ]
-      args << "--enable-64bit" if MacOS.prefer_64_bit?
+      args << "--enable-macos-target=#{MacOS.version}" if OS.mac?
+      args << "--enable-64bit" if MacOS.prefer_64_bit? || OS.linux?
       system "./configure", *args
       # Remove the broken (for anyone but Firefox) install_name
-      inreplace "config/autoconf.mk", "-install_name @executable_path/$@ ", "-install_name #{lib}/$@ "
+      inreplace "config/autoconf.mk", "-install_name @executable_path/$@ ", "-install_name #{lib}/$@ " if OS.mac?
```

Comment:
This chunk sadly is necessary, otherwise the `inreplace` fails and stops the build.